interfaces/cython/cantera/examples/onedim/diffusion_flame.py

```diff
@@ -22,7 +22,7 @@
 
 # Create the gas object used to evaluate all thermodynamic, kinetic, and
 # transport properties.
-gas = ct.Solution('gri30.xml', 'gri30_mix')
+gas = ct.Solution('gri30.yaml', 'gri30')
```

Comment:
Might as well drop the phase name here, since there's only one. Likewise in the other examples.